src/libraries/System.Net.Quic/src/System/Net/Quic/QuicConnectionOptions.cs

```diff
@@ -124,6 +139,14 @@ public QuicReceiveWindowSizes InitialReceiveWindowSizes
     /// </summary>
     public TimeSpan HandshakeTimeout { get; set; } = QuicDefaults.HandshakeTimeout;
 
+    /// <summary>
+    /// Optional callback that is invoked when new stream limit is released by the peer. Corresponds to receiving MAX_STREAMS frame.
```

Comment:
```suggestion
    /// Optional callback that is invoked when a new stream limit is released by the peer. Corresponds to receiving a STREAMS_AVAILABLE frame.
```

Reply:
It is MAX_STREAM: https://www.rfc-editor.org/rfc/rfc9000.html#name-max_streams-frames

Reply:
Added the article though.